chaos_genius/core/rca/rca_utils/api_utils.py

```diff
@@ -116,10 +116,15 @@ def kpi_line_data(kpi_id):
             raise ValueError("No data found.")
 
         final_data = data_point.data
-        for row in final_data:
-            row["date"] = convert_datetime_to_timestamp(
-                get_rca_date_from_string(row["date"])
-            )
+        if not download:
+            for row in final_data:
+                row["date"] = convert_datetime_to_timestamp(
+                    get_rca_date_from_string(row["date"])
+                )
+        else:
+            for row in final_data:
+                row["date"] = get_rca_date_from_string(row["date"])
+
```

Comment:
Why is this converted only when downloading?

Reply:
The timestamp generated by `convert_datetime_to_timestamp` was not usable as creation of a datetime object from that timestamp failed. So just for downloads the result is returned without the use of the said function.